src/pages/billing/List.js

```diff
@@ -245,14 +243,12 @@ export const BillingList = ({ queryParams: { selectedName } }) => {
     Utils.withBusyState(setIsLoadingAccounts)
   )(async () => {
     if (Auth.hasBillingScope()) {
-      setBillingAccounts(await Ajax(signal).Billing.listAccounts())
+      const accounts = await Ajax(signal).Billing.listAccounts()
+      const byAccountName = Object.fromEntries(_.map(acc => [acc.accountName, acc], accounts))
```

Comment:
`reduce` is my go-to pattern for this, but yours reads nicely so feel free to ignore.
```suggestion
      const accountsByName = _.reduce((acc, x) => {...acc, [x.accountName]: x}, {}, accounts))
```

Reply:
Indeed. I was hoping to not have to hand-roll this myself, though. lodash's `fromEntries` follows your suggestion almost exactly.
I was looking for some function that would "index" this a list with a key-function 
```javascript
listAccounts().then(index('accountName')).then(setBillingAccounts)
```

Reply:
I'm afraid I do not. Perhaps this function belongs in our Utils.

Reply:
I think you might have been looking for [_.fromPairs](https://lodash.com/docs/4.17.15#fromPairs) here; however, the simplest way to do this would be [_.keyBy](https://lodash.com/docs/4.17.15#keyBy):
```js
const accountsByName = _.keyBy('accountName', accounts)
```